README.md

```diff
@@ -12,37 +12,75 @@ products:
 
 # Quickstart - JavaScript SDK v10 for Azure Storage
 
-This repository implements the [quickstart](http://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-nodejs-v10) sample for the [Azure Storage v10 SDK for JavaScript](https://github.com/Azure/azure-storage-js).
+This repository implements the [Quickstart](http://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-nodejs-v10) sample for the [Azure Storage v10 SDK for JavaScript](https://github.com/Azure/azure-storage-js).
+
+# SDK Versions
+You will find the following folders: azure-storage-js-v10-quickstart-v10, which references the version 10.0 SDK and azure-storage-js-v10-quickstart-v12, which uses the 12.0 version of the SDK.
```

Comment:
See my format recommendation from the key-vault sample and use same here.  Thanks.